core/src/main/java/io/github/wysohn/triggerreactor/core/script/interpreter/Interpreter.java

```diff
@@ -263,6 +265,57 @@ private void start(Node node) throws InterpreterException {
                     throw new InterpreterException("Unexpected token for IF statement! -- " + resultToken);
                 }
             }
+        } else if ("TRY".equals(node.getToken().value)) {
+            List<Object> statements = node.getChildren()
+                    .stream()
+                    .map(s -> s.getToken().value)
+                    .collect(Collectors.toList());
+
+            if (node.getChildren().size() == 2) {
+                if (statements.contains("<CATCH BODY>")) {
+                    String varName = node
+                            .getChildren().get(1)
+                            .getChildren().get(0)
+                            .getChildren().get(1)
+                            .getToken()
+                            .value
+                            .toString();
+
+                    try {
+                        start(node.getChildren().get(0));
+                    } catch (Throwable e) {
+                        getVars().put(varName, e);
+                        start(node.getChildren().get(1));
+                    }
+                } else {
+                    try {
+                        start(node.getChildren().get(0));
+                    } finally {
+                        start(node.getChildren().get(1));
+                    }
+                }
+            } else if (node.getChildren().size() == 3) {
+                String varName = node
```

Comment:
Love it! Thanks for the hard work.

Though, does the unit test covers all the newly added lines? I'm a little bit worried about this part particularly because it is a long, nested call.

You can check that by using the test with coverage function if you are using IntelliJ (https://www.jetbrains.com/help/idea/code-coverage.html)

Or other IDE should have similar functionality for sure